sdk/face/ai-vision-face-rest/test/public/longRunningOperation.spec.ts

```diff
@@ -14,114 +14,158 @@ const randomUUID = (): string =>
   );
 
 describe("PersonDirectory", () => {
-    let recorder: Recorder;
-    let client: FaceClient;
-  
-    beforeEach(async (context) => {
-      recorder = await createRecorder(context);
-      client = await createClient(recorder);
+  let recorder: Recorder;
+  let client: FaceClient;
+
+  beforeEach(async (context) => {
+    recorder = await createRecorder(context);
+    client = await createClient(recorder);
+  });
+
+  afterEach(async () => {
+    if (recorder?.recordingId) {
+      await recorder.stop();
+    }
+  });
+
+  it("CreateDynamicPersonGroup", async () => {
+    const dynamicPersonGroupId = recorder.variable("dynamicPersonGroupId", randomUUID());
+    const createDynamicPersonGroupResponse = await client
+      .path("/dynamicpersongroups/{dynamicPersonGroupId}", dynamicPersonGroupId)
+      .put({
+        body: { name: dynamicPersonGroupId },
+      });
+    assert.equal(createDynamicPersonGroupResponse.status, "200");
+
+    const deleteDynamicPersonGroupResponse = await client
+      .path("/dynamicpersongroups/{dynamicPersonGroupId}", dynamicPersonGroupId)
+      .delete();
+    const deleteDynamicPersonGroupPoller = await getLongRunningPoller(
+      client,
+      deleteDynamicPersonGroupResponse,
+    );
+    await deleteDynamicPersonGroupPoller.pollUntilDone();
+    assert.equal(deleteDynamicPersonGroupPoller.getOperationState().status, "succeeded");
+  });
+
+  it("CreateDynamicPersonGroupWithPerson", async () => {
+    const createPersonResponse = await client.path("/persons").post({
+      body: { name: "test" },
     });
-  
-    afterEach(async () => {
-      if (recorder?.recordingId) {
-        await recorder.stop();
-      }
+    if (isUnexpected(createPersonResponse)) {
+      throw new Error(createPersonResponse.body.error.message);
+    }
+    const { personId } = createPersonResponse.body;
+
+    const dynamicPersonGroupId = recorder.variable("dynamicPersonGroupId", randomUUID());
+    const createDynamicPersonGroupResponse = await client
+      .path("/dynamicpersongroups/{dynamicPersonGroupId}", dynamicPersonGroupId)
+      .put({
+        body: {
+          name: dynamicPersonGroupId,
+          addPersonIds: [personId],
+        },
+      });
+    assert.equal(createDynamicPersonGroupResponse.status, "202");
+
+    const createPersonPoller = await getLongRunningPoller(client, createPersonResponse);
+    await createPersonPoller.pollUntilDone();
```

Comment:
Ususally we could refer the final result like this

```
const result = await createPersonPoller.pollUntilDone();
```

Reply:
After taking a look into this, I decide to keep the current code `poller.getOperationState().status` because:

1. Somehow the return types of `pollUntilDone` only have a `status` field and no `body`, so there is no useful information.
2. The return value of `pollUntilDone` is actually a response to a GET request, but what I need is the final status of the LRO.

So, I’m not going to change code here, but will investigate why the generated return types from our spec are so weird.

Reply:
We do have some known issue for our DynamicPersonGroup LRO since it's not following the standard.
https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#long-running-operations
I believe the 2nd item you mentioned should be by design, while I'm not quite sure about 1st item.